tests/saitests/sai_qos_tests.py

```diff
@@ -60,6 +61,41 @@
 ECN_INDEX_IN_HEADER = 53 # Fits the ptf hex_dump_buffer() parse function
```

Comment:
index in which header?

Reply:
This is in the packet returned by the PTF framework. The packet starts with L2 header.